options.ts

```diff
@@ -745,6 +745,8 @@ const options = mkOptions(OPTIONS, {
 
     notifications: {
         position: opt<"top" | "top right" | "top left" | "bottom" | "bottom right" | "bottom left">("top right"),
+        monitor: opt(0),
+        active_monitor: opt(true), //unsure if I should make the default true/false. I use false, but OSD's version has default as true.
```

Comment:
```suggestion
        active_monitor: opt(true),
```
Yes, active monitor as default is ok. User's can disable if desired.